### ISSUE TITLE
Documentation: Kubernetes (Docker for Mac) Skipper and Data Flow Server Ports

### ISSUE THREAD

**Issue by jkupcho** [spring-cloud/spring-cloud-dataflow#3115]
**Description:**
I'm using Docker for Mac's install of Kuberentes, and following the Kubernetes Install Documentation for `v2.0.1.RELEASE`.  When reaching the final portion of the install when you actually install the Data Flow Server, both the Data Flow Server and Skipper try to expose via port `80`.

**Release versions:**
```
HTTP/1.1 200
Content-Type: application/json;charset=UTF-8
Transfer-Encoding: chunked
Date: Fri, 29 Mar 2019 14:12:51 GMT

{"featureInfo":{"analyticsEnabled":true,"streamsEnabled":true,"tasksEnabled":true,"schedulerEnabled":true,"grafanaEnabled":true},"versionInfo":{"implementation":{"name":"spring-cloud-dataflow-server","version":"2.0.1.RELEASE"},"core":{"name":"Spring Cloud Data Flow Core","version":"2.0.1.RELEASE"},"dashboard":{"name":"Spring Cloud Dataflow UI","version":"2.0.1.RELEASE"},"shell":{"name":"Spring Cloud Data Flow Shell","version":"2.0.1.RELEASE","url":"https://repo.spring.io/libs-release/org/springframework/cloud/spring-cloud-dataflow-shell/2.0.1.RELEASE/spring-cloud-dataflow-shell-2.0.1.RELEASE.jar"}},"securityInfo":{"authenticationEnabled":false,"authenticated":false,"username":null,"roles":[]},"runtimeEnvironment":{"appDeployer":{"deployerImplementationVersion":"2.0.0.RELEASE","deployerName":"Spring Cloud Skipper Server","deployerSpiVersion":"2.0.0.RELEASE","javaVersion":"1.8.0_192","platformApiVersion":"","platformClientVersion":"","platformHostVersion":"","platformSpecificInfo":{"default":"kubernetes"},"platformType":"Skipper Managed","springBootVersion":"2.1.3.RELEASE","springVersion":"5.1.5.RELEASE"},"taskLaunchers":[{"deployerImplementationVersion":"2.0.0.RELEASE","deployerName":"KubernetesTaskLauncher","deployerSpiVersion":"2.0.0.RELEASE","javaVersion":"1.8.0_192","platformApiVersion":"v1","platformClientVersion":"unknown","platformHostVersion":"unknown","platformSpecificInfo":{"namespace":"default","master-url":"https://10.96.0.1:443/"},"platformType":"Kubernetes","springBootVersion":"2.1.3.RELEASE","springVersion":"5.1.5.RELEASE"}]},"grafanaInfo":{"url":"http://grafana:3000","refreshInterval":15},"_links":{"self":{"href":"http://localhost:8880/about"}}}
```

**Custom apps:**
N/A

**Steps to reproduce:**
* Run Kubernetes via Docker for Mac
* Step through the documentation from `5. Getting Started - Kubernetes` 

**Screenshots:**
N/A

**Additional context:**
There's a note for Minikube users stating you can expose the Internal Endpoint URL via `NodePort`.  Maybe include an example on how to override for the exposed target port if you're running via Docker for Mac.  For example, I overrode the port to 8880.

**Comment by sabbyanandan** [comment]
Thanks, @jkupcho. We will have a look. Just curious: have you tried the SCDF helm-chart on Docker for Mac?

**Comment by jkupcho** [comment]
I haven’t, actually I can give it a go for you and note the results.

**Comment by jkupcho** [comment]
So just ran the helm chart, skipper doesn't expose itself externally, only the Spring Cloud Config Server.  So the two installs seem to do different things.

**Comment by sabbyanandan** [comment]
Hi, @jkupcho. My apologies for the delay. I finally got around to try the Docker for Desktop. 

(1) Skipper is supposed to be an internally-only backing resource for SCDF (to provide CD/versioning capabilities); in other words, it doesn't have to be exposed as an externally accessible resource. SCDF can rely on it via an Internal IP similar to a typical production install. We will look into it hiding it; perhaps also change the default port (i.e., 7577/80) to something else, which SCDF would already know given how they are configured.

(2) Same reason as above. That's why the helm-chart doesn't expose Skipper as an externally accessible service. I'm glad there are no other gotchas on helm-chart + Docker for Desktop.

(3) As for Prom/Influx + Graf in the helm-chart, that is something we could do in the future. We recently discussed the option and the design approach at [spring-cloud/spring-cloud-dataflow/issues/2689#issuecomment-472520678](https://github.com/spring-cloud/spring-cloud-dataflow/issues/2689#issuecomment-472520678). That's also an open area for the community to contribute. 😄 

(4) We will fix `kubectl get svc <helm-release-name>-data-flow-server` step in the docs when we fix (1), as part of this issue.

(5) We have resolved the docs issue on the `wget` command recently. That should be showing up correctly in [v2.0.2.BUILD-SNAPSHOT](http://docs.spring.io/spring-cloud-dataflow/docs/2.0.2.BUILD-SNAPSHOT/reference/htmlsingle/#getting-started) docs.
